ctags.go

```diff
@@ -222,37 +221,27 @@ func (p *ctagsProcess) Parse(name string, content []byte) ([]*Entry, error) {
 	}
 
 	if !utf8.Valid(content) {
-		if p.Info != nil {
-			p.Info.Printf("ctags skipping file due not being utf-8 encoded: %s", name)
-		}
-		return nil, nil
+		return nil, newParseError("file is not utf-8 encoded")
```

Comment:
I decided to omit the file name from these errors, since `Parse` is called with a single file and the caller can easily correlate the error with a specific file.